intuita-webview/src/shared/Snippet/detectTheme.ts

```diff
@@ -0,0 +1,11 @@
+export function detectBaseTheme(): 'vs-light' | 'vs-dark' {
+	const attribute = document.body.getAttribute('data-vscode-theme-kind');
+	if (
+		attribute &&
```

Comment:
I think we can then drop `attribute &&` am I right?

Reply:
Yeah that is true